### ISSUE TITLE
Questions to be addressed 

### ISSUE THREAD

**Issue by jread-usgs** [USGS-VIZLAB/hurricane-maria#1]
1. Do we have adequate flood stage info? If not, can we use > 90% (and explain that in plain english)
2. what geographic info are we including for reference? big cities/capital, county equivalent
3. use custom (higher res) PR shapefile? where from? give appropriate credit
4. make sure we link to the appropriate WSC in the text
5. precip data is more raw than NOAA stage IV. What QAQC (if any) do we apply?
6. which "experiences" are supported? Possibilities: landscape, portrait, portrait-mobile, ...others?
7. how to animate scenes w/o dogging the mobile browser? As simple as using `requestAnimationFrame` instead of `setTimeout` for animation in JS? Do we need to animate raster instead of svg for precip?
8. export a high-value (and high-res) gif as product, or based on screen capture?

**Comment by jiwalker-usgs** [comment]
I looked at `requestAnimationFrame` over the weekend, it is definitely doable to switch to that, it may result in missing some frames but will provide a constant speed across devices.

**Comment by mbucknell** [comment]
Other benefits of requestAnimationFrame it stops when the user switches to a different tab, so won't dog the device's CPU when not visible.

**Comment by dblodgett-usgs** [comment]
1) We DO have flood stage data. "Action", "Flood", "Moderate", and "Major".
5) I found precip that is the NOAA Stage IV, but it's in a super goofy format... not going to be THAT hard to deal with though.
